libraries/AP_VisualOdom/AP_VisualOdom.cpp

```diff
@@ -90,6 +90,22 @@ const AP_Param::GroupInfo AP_VisualOdom::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("_VEL_M_NSE", 5, AP_VisualOdom, _vel_noise, 0.1),
 
+    // @Param: _POS_M_NSE
+    // @DisplayName: Visual odometry position measurement noise
+    // @Description: Visual odometry position measurement noise in meters
```

Comment:
I think that maybe we should change this wording to make it clear that this is the minimum noise value sent to the EKF.  Perhaps make the Description, "Visual odometry position measurement noise minimum.  This value will be used if the sensor provides a lower noise value (or not noise value)"

Reply:
Yes. It is more clear. Thank you